apps/mobile/metro-now/codegen.sh

```
@@ -0,0 +1,3 @@
+./apollo-ios-cli fetch-schema
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)